.editorconfig

```diff
@@ -73,3 +73,6 @@ csharp_style_unused_value_expression_statement_preference = discard_variable:non
 
 # IDE0065
 csharp_using_directive_placement = outside_namespace
+
+# xUnit1004: Test methods should not be skipped
+dotnet_diagnostic.xUnit1004.severity = none
```

Comment:
⚠️ This should be set to `refactoring` instead of `none` to allow users to still use the code fix in the IDE to unskip tests (especially applies to cases where you want to **Fix All in Document**).